tools/ITER_Profile_reader.py

```diff
@@ -0,0 +1,66 @@
+# This class provides routines for reading input profiles from CORSICA data files
+
+import numpy as np
+
+class ITER_Profile_reader:
+
+    def __init__(self, filename):
+        self.filename = filename
+        
+        f = open(self.filename,'r')
```

Comment:
I would recommend putting the remainder of the constructor in a separate method which could be called again if necessary.